src/xmlwrapp.mk

```diff
@@ -19,10 +17,8 @@ endef
 define $(PKG)_BUILD
     cd '$(1)' && ./configure \
         --host='$(TARGET)' \
-        --disable-shared \
+        $(MXE_CONFIGURE_OPTS) \
         --prefix='$(PREFIX)/$(TARGET)' \
```

Comment:
remove `--host` and `--prefix` when using `$(MXE_CONFIGURE_OPTS)`

Reply:
Ok